### PR TITLE
Add more base data for expression analysis

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR adds some more fundamental data for expression data analysis. In my [first module PR](https://github.com/nf-core/modules/blob/a8af48623438d363fec7ea25fb17dd8024bd374b/tests/modules/nf-core/deseq2/differential/main.nf) I've 'spoofed' some of this for the purposes of testing, but I've realised that I'd have to do that repeatedly in other PRs, so better if we just store the inputs in a static way.

### CODE REVIEW

File: data/genomics/mus_musculus/rnaseq_expression/README.md [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+ * The value of the sample sheet contrast column used to identify reference samples.
+ * The value of the sample sheet contrast column used to identify target/ treatment samples.
+ * A comma-separated list of other columns to use as blocking variables (can empty)
+
+### SRP254919.salmon.merged.gene_counts.top1000cov.tsv
+
+Count matrix derived by running [rnaseq](https://nf-co.re/rnaseq) against the mouse genome GRCm38 with default parameters. The top 1000 most variable genes were selected from `salmon.merged.gene_counts` in R as follows: 
 
 ```
 expression <- read.table("salmon.merged.gene_counts.tsv")
```
Comment (mashehu) [inline data/genomics/mus_musculus/rnaseq_expression/README.md:29]:
```suggestion

```r
expression <- read.table("salmon.merged.gene_counts.tsv")
```

File: data/genomics/mus_musculus/rnaseq_expression/SRP254919.gene_meta.tsv [hunk 2]
```diff
@@ -0,0 +1,1001 @@
+gene_id	gene_name
+ENSMUSG00000000305	Cdh4
+ENSMUSG00000000308	Ckmt1
```
Comment (mashehu) [inline data/genomics/mus_musculus/rnaseq_expression/SRP254919.gene_meta.tsv:3]: Could we have a shorter list for this?

**Review by mashehu — APPROVED** [review]

File: data/genomics/mus_musculus/rnaseq_expression/SRP254919.gene_meta.tsv (hunk 2 above)
Comment (pinin4fjords) [reply data/genomics/mus_musculus/rnaseq_expression/SRP254919.gene_meta.tsv:3]: This is designed to match the expression matrix. It's for use in downstream applications that process that matrix alongside the row (gene) and column (sample metadata). I can 'fake it' in each application, but then I have to keep repeating the faking.

File: data/genomics/mus_musculus/rnaseq_expression/README.md (hunk 1 above)
Comment (pinin4fjords) [reply data/genomics/mus_musculus/rnaseq_expression/README.md:29]: Thanks!

**Pull request merged**